discovery-infra/test_infra/controllers/node_controllers/vsphere_controller.py

```diff
@@ -0,0 +1,213 @@
+import time
+from builtins import list
+from typing import Tuple, List, Callable, Any
+
+import libvirt
+from pyVim import task
+from pyVim.connect import SmartConnect, Disconnect
+from pyVmomi import vim
+
+from test_infra.controllers.node_controllers.disk import Disk
```

Comment:
Import from the package after declaring it in the __init__ file.
You will be able to merge imports:

```suggestion
from test_infra.controllers.node_controllers import Disk, Node, NodeController ..
```